js/screen/app/partials/_main_view.html

```diff
@@ -81,6 +83,31 @@ <h2 class="page-header">Controller</h2>
     </div>
   </div>
 
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="page-header">Saving player state</h2>
```

Comment:
This is the wrong heading, isn't it?

```suggestion
      <h2 class="page-header">Localisation</h2>
```

Reply:
✅